docs/manage/_get_support.rst

```diff
@@ -60,6 +60,18 @@ About IP addresses
 * ``http://192.168.1.1:8080`` means: "Connect to IP address ``192.168.1.1`` on port ``8080`` with the HTTP protocol". The browser will the continue to try to reach this address, but may fail for instance if Kolibri isn't running, or if a step along the way blocks access.
 
 
+.. _firewalls:
+
+Firewalls (Windows)
```

Comment:
Nitpicking: isn't there just **a** (one) Windows firewall built in, and enabled by default? Why plural (here and below)?

Reply:
Mostly because in the last paragraph you mention disabling **just one**, so implying there is more than one might confuse the user (_"How do I (temporarily) disable the others?"_).

Reply:
yeah .. I think the confusion is whether this applies only to Windows or not

Is this section about "the firewall on windows" or is it about "firewalls which may or may not be installed on your operating system"

Maybe we should make it the latter?

Reply:
I just wanna catch the attention of especially Windows users with the headline. I know that other systems may have Firewalls as well, but they have to be installed.

For Windows, the Firewall is often the one that is built-in, but OEM pre-installed versions of Windows might also have some other third-party Firewall, or is that really rare?

Reply:
okay, so just call it 'Windows Firewall'?

Reply:
But then it would seem entirely irrelevant to non-Windows users.. "Firewall (Windows)" was intentional :)

Reply:
'Firewall (Windows)' and 'Windows Firewall' both seem equally specific to Windows to me...

`¯\_(ツ)_/¯`